tests/jballerina-unit-test/src/test/java/org/ballerinalang/nativeimpl/jvm/tests/StaticMethods.java

```diff
@@ -173,6 +174,10 @@ public static Object acceptNothingInvalidAnyReturn() {
         return "invalid java string";
     }
 
+    public static List<String> acceptNothingInvalidAnydataReturn() {
```

Comment:
shall we add a test which return unions of anydata ( int|string[]) and return int[] )